### ISSUE TITLE
Bio.PDB.Residue raises AssertionError for two alternative 'shifted' conformations

### ISSUE THREAD

**Issue by touwwouter** [biopython/biopython#302]
An AssertionError is raised by StructureBuilder/Residue for two alternative 'shifted' polypeptide stretches (4aph chain P). In the 4aph paper (http://dx.doi.org/10.1038/srep00717) the authors describe that they observed two different conformation of the peptides, shifted by one residue. In the first conformation residues 1 to 6 were fitted (Asp-Arg-Val-Tyr-Ile-His). In the second conformation residues 2-7 (Arg-Val-Tyr-Ile-His-Pro) were fitted. Both conformations were modeled with 50% occupancy.

Under normal circumstances the AssertionError makes sense, but I would like to continue creating a Bio.PDB.Structure for entries like 4aph without catching the AssertionError. I hope this issue can be fixed.

```
Biopython version 1.63:
-----------------------
>>> from Bio import PDB       
>>> p = PDB.PDBParser()
>>> s = p.get_structure("4aph", "pdb4aph.ent")
/usr/lib/pymodules/python2.7/Bio/PDB/StructureBuilder.py:121: PDBConstructionWarning: WARNING: Residue (' ', 2, ' ') redefined at line 10334.
  PDBConstructionWarning)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/pymodules/python2.7/Bio/PDB/PDBParser.py", line 82, in get_structure
    self._parse(handle.readlines())
  File "/usr/lib/pymodules/python2.7/Bio/PDB/PDBParser.py", line 108, in _parse
    self.trailer=self._parse_coordinates(coords_trailer)
  File "/usr/lib/pymodules/python2.7/Bio/PDB/PDBParser.py", line 219, in _parse_coordinates
    structure_builder.init_residue(resname, hetero_flag, resseq, icode)
  File "/usr/lib/pymodules/python2.7/Bio/PDB/StructureBuilder.py", line 153, in init_residue
    disordered_residue.disordered_add(new_residue)
  File "/usr/lib/pymodules/python2.7/Bio/PDB/Residue.py", line 160, in disordered_add
    assert(not self.disordered_has_id(resname))
AssertionError

```

**Comment by chebizarro** [comment]
It looks like a fix for this issue was merged with commit a6c8e9b, can this issue be closed now?

**Comment by peterjc** [comment]
I think you're right Chris, thanks - closing.
